### ISSUE TITLE
MongoDB connection

### ISSUE THREAD

**Issue by ibra** [BotMyst/BotMystRevival#16]

**Comment by FadTheChad** [comment]
I know this issue was made pretty while ago B U T, since the bot switched to ts, it would be nice seeing it use `@typegoose/typegoose` for db (if you're still thinking of the mongoose db route)

Typegoose is basically a ts wrapper for mongoose which doesn't modify the original functions of mongoose, but gives you a better hold over types when using Schemas

**Comment by ibra** [comment]
yeah that is the plan, will try work on it when I get free from exams

**Comment by FadTheChad** [comment]
Epic. Good luck!
